### ISSUE TITLE
Is webm still supported?

### ISSUE THREAD

**Issue by AnimatorTao** [Sagnac/streamsave#9]
force_extension=.webm gives me a "Cache dumping stopped due to a error" message with the output being a 1kb webm file.

**Comment by Sagnac** [comment]
Yes, keep in mind only VP8 and VP9 are part of the WebM spec. Can you share the format information of the stream you're trying to save?

**Comment by AnimatorTao** [comment]
Yeah that was the problem, thought was still vp9 but they've apparently changed it.
Another thing, since there's not encoding involved I guess the "keeping active tracks" option like muted, hidden subs is not available and I'm better off just ffmpeg the outputs?

**Comment by Sagnac** [comment]
Sorry, could you rephrase your question? I'm not sure I understand. Whether the audio is muted or not makes no difference, only which track is selected. If the stream has other tracks you want written you could switch to them and when they're properly cached you could write them to file.

Although if that's your objective (including multiple tracks) it's probably better to automate this using yt-dlp directly and using a tool like mkvmerge without having to encode.

**Comment by AnimatorTao** [comment]
Nothing to do with multiple tracks, poor choice of words on my part sorry.
>Whether the audio is muted or not makes no difference
This was my question, if I mute the player or not before dumping the output will still have audio?

**Comment by Sagnac** [comment]
Yes, as long as that track is selected and cached it will be present in the output.

**Comment by AnimatorTao** [comment]
Ok thanks for the answer.
